packages/react-components/src/select/stories/standard.stories.mdx

```diff
@@ -35,6 +35,10 @@ A select can be used to select between choices.
 
 ## Usage
 
+<WarningMessage>
+    This component have the same feature set as the <a href="?path=/docs/components-select-standard--default-story">standard select</a> component.
+</WarningMessage>
```

Comment:
Not sure about this. You are using in the docs of the standard select to look at standard select docs ;)

Reply:
bad copy pasta